### DISCUSSION TITLE
Creating first Plugin

### DISCUSSION THREAD

**Discussion by cntabana** [awcodes/hydro#1] [Q&A]
Hi, first of all, thank you for your effort in building this package.

I'm trying to create the first skeleton, but I received this error message. 

Cannot assign null to property App\Actions\ProcessPluginStubs::$vendorNamespace of type string

Do you think I'm missing something?

<img width="923" alt="Screenshot 2023-03-08 at 13 24 50" src="https://user-images.githubusercontent.com/6320126/223725881-750d53dd-4471-4749-8efd-809274e7430b.png">

Thank you

**Comment by awcodes** [comment]
Everything looks ok. You have a namespace defined. I'll see if I can replicate and get back to you.

**Comment by awcodes** [comment]
Ok. So, there's a bug that I'll get fixed as soon as I can. But if you provide the `--vendor-namespace=Andy` flag as part of the new command it should work as expected.

Sorry about that.

**Comment by cntabana** [comment]
Thank you , let me try it.

**Comment by cntabana** [comment]
It works now. Thank you

**Comment by awcodes** [comment]
This should be fixed with v0.1.1. Thanks again for reporting it.
